### ISSUE TITLE
[verbs] compiler warning

### ISSUE THREAD

**Issue by jsquyres** [ofiwg/libfabric#1837]
Got this compiler warning from the verbs provider this morning:

```
In file included from prov/verbs/src/verbs_cq.c:35:0:
prov/verbs/src/verbs_cq.c: In function ‘fi_ibv_cq_readerr’:
./include/fi_mem.h:198:2: warning: ‘slist_entry’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  slist_insert_head(&util_buf->entry, &pool->buf_list);
  ^
prov/verbs/src/verbs_cq.c:46:22: note: ‘slist_entry’ was declared here
  struct slist_entry *slist_entry;
                      ^
```

**Comment by a-ilango** [comment]
Interestingly, I don't see this warning in my build. Let me take a look at this. --enable-debug option should turn on -Wall, right?

**Comment by jsquyres** [comment]
Yes `--enable-debug` turns on all of these:

```
CFLAGS="$CFLAGS -g -O0 -Wall -Wextra -Wundef -Wno-unused-parameter -Wno-sign-compare -Wno-missing-field-initializers"
```

**Comment by jsquyres** [comment]
Looks like you files #1845 to fix this.

Pro Github tip: if you put "Fixes #1837" in the commit message (or even the description of the PR, I think?), it'll auto-close this issue once that PR is merged.

**Comment by a-ilango** [comment]
Yes. Nice, that's an useful tip. Let me try adding it in the description now.
